cpu/samd5x/cpu.c

```diff
@@ -386,6 +386,11 @@ void cpu_init(void)
     /* trigger static peripheral initialization */
     periph_init();
 
+#ifdef MODULE_PERIPH_GPIO_TAMPER_WAKE
+    /* initialize tamper detection */
+    rtc_tamper_init();
+#endif
```

Comment:
Why not have it called in `periph_init()`?

Reply:
Moved it there.

Reply:
Does this feature exist on other MCUs ?  Otherwise why move it to periph_init() ?

Reply:
It's also on saml1x but there we don't need it because the CPU does not support hibernate and EXTI is still active in STANDBY.

We might be able to save some power by disabling EXTI in STANDBY and using RTC instead.

Reply:
That's an interesting idea but I personally think it's a shame to pollute `periph_init()` with SAM0 stuff.

Reply:
I can move it back to `cpu.c` then.

Reply:
Leave it there for now. We'll use XFA when available.